homeworks/hw1.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+let x = 5;
+let y = 10;
+
+let addition = x + y;
+let subtraction = y - x;
+let multiplication = x * y;
+let division = y / x;
+let exponentiation = x ** y;
+
+console.log('Додавання: ' + addition);
```

Comment:
Тут можна використати сучасніший синтаксис і записати так
```suggestion
console.log(`Додавання:  ${addition}`);
```

Reply:
Updated.